packages/@aws-cdk/aws-apigateway/lib/authorizers/lambda.ts

```diff
@@ -190,7 +189,7 @@ export interface RequestAuthorizerProps extends LambdaAuthorizerProps {
    *
    * @see https://docs.aws.amazon.com/apigateway/api-reference/link-relation/authorizer-create/#identitySource
    */
-  readonly identitySources: IdentitySource[];
+  readonly identitySources: string[];
```

Comment:
Why isn't this `IdentitySource`? It would make sense, no? What am I missing?

Reply:
The `IdentitySource` class is simply providing helper functions to build different types of identity source strings. 

The type here being `string[]` allows for a user to specify a identity source even if a corresponding helper method doesn't exist in `IdentitySource` (i.e., when APIGW adds more and we don't support yet).

It also stays consistent with `TokenAuthorizerProps` which uses `string` type for the `identitySource` property

Reply:
Normally, we would keep this strong-typed and offer something like `IdentitySource.custom("bla")` or `IdentitySource.of("bla")` for future-proofing.

Reply:
Sure, but I would like to keep it consistent with the [`identitySource` property here](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-apigateway/lib/authorizers/lambda.ts#L135) which had already previously shipped and changing the type there would be breaking.

Would just adding a `IdentitySource.custom()` method satisfy? Do you have an different suggestion?

Reply:
I am okay with changing this to a string.